hoomd/filter/custom.py

```diff
@@ -0,0 +1,51 @@
+"""Contains a class for custom particle filters in Python."""
+from abc import ABCMeta, abstractmethod
+
+class CustomFilter(metaclass=ABCMeta):
```

Comment:
Based on the interface this class defines, it might actually make more sense to inherit directly from `collections.abc.Callable` and `collections.abc.Hashable`.